content/en/blog/_posts/2021-11-02-csi-migration-status.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: blog
+title: "Kubernetes 1.23: Kubernetes In-Tree to CSI Volume Migration Status Update"
+date: 2021-11-03T15:00:00+08:00
```

Comment:
This date should be the scheduled publish date of this blog.  You'll need to check with release/doc team on this.

Reply:
I checked with @karenhchu and it seems the release date for it should be 2021-12-13. I am updating the doc to reflect that.